sdk/eventhubs/azure-messaging-eventhubs/README.md

```diff
@@ -0,0 +1,3 @@
+# Azure Messaging EventHub service
```

Comment:
should it live under `sdk/messaging/eventhubs` instead of just `sdk/eventhubs`?

Reply:
Every language except Go has it under sdk/eventhubs.

That also allows us to have a separate CI pipeline for eventhubs and servicebus (when it comes along).